<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Vyhledání místa dle souřadnic</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Zeměpisná délka</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSearch">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Hledej</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLongitude">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>571</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_address">
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Zeměpisná šířka</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLatitude">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonSearch</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>displayDetails()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>displayDetails()</slot>
 </slots>
</ui>
